.github/workflows/ci.yml

```diff
@@ -30,11 +31,11 @@ jobs:
 
           ./.github/scripts/set-output version "${VERSION}"
       - name: Install git-cliff
-        uses: baptiste0928/cargo-install@bf6758885262d0e6f61089a9d8c8790d3ac3368f # v1.3.0
```

Comment:
Curious why this is unpinned? Best practice is to [pin to a commit SHA](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions).

Reply:
@AaronFriel I'm just trying to get the build passing, seems it's been red for a while. I'll change it.

Reply:
@AaronFriel the remaining [failure](https://github.com/pulumi/watchutil-rs/actions/runs/5814345665/job/15766330788) is on the release step, and I'm pretty out of my element. It seems like pu/pu migrated to cosign in https://github.com/pulumi/pulumi/pull/11310, so my plan was to try porting that over to this repo. Would love to hear if you have alternatives before I go down that path.

Reply:
Discussed 1:1, I've disabled the signing step.